docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -160,6 +160,24 @@ The database connection tab shows the command that can be used to connect to the
 modify the database name of the connection and then click the Run button to open a new terminal tab
 with that command executed.
 
+## Connecting to a Windows desktop
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+can also press `Ctrl+Shift+T/Cmd+T` to achieve the same result.
+1. Look for the desktop you wish to connect to and click the Connect button.
+1. Select the user you wish to use and press `Enter`.
+
+A new tab will open with a remote desktop session.
+
+Alternatively, you can look for the desktops in the search bar and press `Enter` to connect to it.
+
+During a remote session, you can share your clipboard and a local directory, if permitted by your role.
+
+<Admonition type="note">
+  Directory sharing in Teleport Connect is more permissive than in the Web UI.
```

Comment:
It'd be nice to link "more permissive than in the Web UI" to the "Prohibited directories" section from `docs/pages/enroll-resources/desktop-access/directory-sharing.mdx`. Otherwise this admonition doesn't tell you much if you don't know about the restrictions in the Web UI.

Matter of fact, we could drop the whole admonition – why mention the lack of restrictions in the first place? I think it's enough that we mention in "Prohibited directories" that this doesn't affect Connect.